src/main/java/tdl/datapoint/video/security/TokenSigningService.java

```diff
@@ -0,0 +1,43 @@
+package tdl.datapoint.video.security;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.crypto.Mac;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.UnsupportedEncodingException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.Base64;
+
+public class TokenSigningService {
```

Comment:
It would be good to have a unit test for this class, so that we capture it's behaviour.

Reply:
Onto it now!

Reply:
Done!